src/modules/post.js

```diff
@@ -0,0 +1,19 @@
+const post = async (id, username, score) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`;
+  const data = {
+    user: username,
+    score,
+  };
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add Contributing and Key Features sections in your readme file to make your readme  descriptive and professional :+1: